index.html

```diff
@@ -380,7 +380,37 @@ <h5 class="modal-title">Other Settings</h5>
             }
         }
 
+        // Checks for any overrides from the querystring and sets the form values as appropriate
+        function checkOverrides() {
+            // Maps between the parameter name and the selector of the input element
+            const configMappings = {
+                "mem": "#inp_memsz",
+                "vgamem": "#inp_vmemsz",
+                "apic": "#inp_apic",
+                "acpi": "#inp_acpi",
+                "pci":  "#inp_pci",
+                "pcivga": "#inp_pcivga",
+                "floppy": "#inp_floppy",
+                "winnt": "#inp_winnt",
+            };
+            
+            const entries = Object.entries(configMappings);
+            for (const [parameter, id] of entries) {
+                var value = getParameterByName(parameter);
+                if (!value)
+                    continue;
+                var input = $(id);
+                // Explict check for checkbox.
+                if (input.prop('type') == "checkbox")
+                    input[0].checked = (value == 1 || value=='') ? true : false;
```

Comment:
You could just set it to `true` here -- the `if(!value)` above already checked for a 'truthy' value, and we shouldn't limit the possibilities here: what if somebody did `?pcivga=yes&apic=absolutely`?